### ISSUE TITLE
Possiblity to add a mail + license key for geekbench 5

### ISSUE THREAD

**Issue by IIPoliII** [masonr/yet-another-bench-script#7]
Hey,
Would it be possible to add a mail + licence key for the geekbench (normally you simply need to add a -r) i did a fork that was working but now it's not the case anymore and i think it could be useful for everyone C: Awesome script by the way !

The format is like that `poli@balal.com Z6POZ-WBGI2-ZX31T-M1241-WRAR7-F1123-DUI2E-LFOL4-6P8ZA`

**Comment by masonr** [comment]
I can probably just add in a check to see if a geekbench key file exists and use that to unlock it, if detected. I'll work on this in the next couple weeks and have you test for me :)

**Comment by IIPoliII** [comment]
Sure! Say me once you implemented it c: will definitely love it

**Comment by masonr** [comment]
Implemented in [testing branch](https://github.com/masonr/yet-another-bench-script/tree/testing). Awaiting user testing before merging changes into master.

To test:
```
echo "email@domain.com 12345-ABCDE-67890-FGHIJ" > geekbench.license
curl -sL "https://raw.githubusercontent.com/masonr/yet-another-bench-script/testing/yabs.sh" | bash -s -- -fi
```

**Comment by masonr** [comment]
Implemented and merged in a748260

No way to automatically add to geekbench account, so you'll have to visit the link that saved to the _geekbench_claim.url_ file.
